tests/layer_tests/tensorflow_tests/test_tf_ReverseV2.py

```diff
@@ -1,56 +1,32 @@
 # Copyright (C) 2018-2022 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-import numpy as np
 import pytest
 from common.tf_layer_test_class import CommonTFLayerTest
 
 
-class TestReverseV2Ops(CommonTFLayerTest):
-    def _prepare_input(self, inputs_dict):
-        for input in inputs_dict.keys():
-            inputs_dict[input] = np.random.random(inputs_dict[input])
-        return inputs_dict
-
-    def create_reversev2_net(self, shape, keep_dims, axis, ir_version):
+class TestReverseV2(CommonTFLayerTest):
+    def create_reverse_v2_net(self, shape, axis):
         import tensorflow as tf
         tf.compat.v1.reset_default_graph()
         with tf.compat.v1.Session() as sess:
-            shapes = shape.copy()
-            if len(shapes) >= 4:
-                shapes.append(shapes.pop(1))
-
-            x = tf.compat.v1.placeholder(tf.float32, shapes, 'Input')
+            x = tf.compat.v1.placeholder(tf.float32, shape, 'Input')
             tf.compat.v1.reverse_v2(x, axis)
             tf.compat.v1.global_variables_initializer()
             tf_net = sess.graph_def
 
         return tf_net, None
 
-    test_data = []
-    test_data.extend([
+    test_data_basic = [
         dict(shape=[5], axis=[0]),
-        pytest.param(dict(shape=[2, 3], axis=[1]), marks=pytest.mark.precommit_tf_fe),
+        dict(shape=[2, 3], axis=[1]),
```

Comment:
Ccould you also add `rank = 1` and `axis = -1` case?

Reply:
Added